hbase-common/src/test/java/org/apache/hadoop/hbase/ClassTestFinder.java

```diff
@@ -46,6 +48,16 @@ public static Class<?>[] getCategoryAnnotations(Class<?> c) {
     return new Class<?>[0];
   }
 
+  public static String[] getTagAnnotations(Class<?> c) {
+    // TODO handle optional Tags annotation
+    Tag[] tags = c.getAnnotationsByType(Tag.class);
```

Comment:
What is this used for?

Reply:
This is the replacement for @Category in Junit 5.

Reply:
The tests is for checking that all test classes have some kind of @Category (or now @Tag) annotation.

Reply:
OK, also comment on another PR, we'd better introduce String constants for these tags.

Reply:
I tried, but String constants are not accepted, only literals.

Reply:
Correction: it works as long as it's a simple static string.
I tried to initialize it with Class.toString() that one did not work.

Reply:
Going to open a new ticket and PR